tests/apollo/util/bft.py

```diff
@@ -692,12 +706,16 @@ def force_quorum_including_replica(self, replica_id, primary=0):
 
 
     async def num_of_slow_path(self):
-        nb_slow_path = 0
-        with trio.move_on_after(seconds=.5):
-            try:
-                metric_key = ['replica', 'Counters', 'slowPathCount']
-                nb_slow_path = await self.metrics.get(0, *metric_key)
-            except KeyError:
-                # metrics not yet available, continue looping
-                pass
-        return nb_slow_path
+        """
+        Returns the total number of requests processed on the slow commit path
+        """
+        with trio.fail_after(seconds=5):
+            while True:
+                with trio.move_on_after(seconds=.5):
+                    try:
+                        metric_key = ['replica', 'Counters', 'slowPathCount']
+                        nb_slow_path = await self.metrics.get(0, *metric_key)
+                        return nb_slow_path
+                    except KeyError:
+                        # metrics not yet available, continue looping
+                        pass
```

Comment:
Nit: I guess pass is interchangeable with continue here. I think continue might be clearer, but it doesn't really matter.

Reply:
Yes, continue would be a bit better - but it's equivalent in this case. I'd prefer not to change it right now.